Formula/scala.rb

```diff
@@ -1,10 +1,8 @@
 class Scala < Formula
   desc "JVM-based programming language"
   homepage "https://www.scala-lang.org/"
-  url "https://downloads.lightbend.com/scala/2.13.8/scala-2.13.8.tgz"
-  mirror "https://www.scala-lang.org/files/archive/scala-2.13.8.tgz"
-  mirror "https://downloads.typesafe.com/scala/2.13.8/scala-2.13.8.tgz"
-  sha256 "2cb31d8469c651839f0e9c837a1ab06550d031726752f54906be1b9de01314cf"
+  url "https://github.com/lampepfl/dotty/releases/download/3.1.3/scala3-3.1.3.tar.gz"
+  sha256 "9e1eefdcab77b2d2a9057b3d6f78301591e9c27513c92413f3c353a77093f2d7"
   license "Apache-2.0"
 
   livecheck do
```

Comment:
We can check https://www.scala-lang.org/download/ for the latest version, or use github releases